DROD/Chat.cpp

```diff
@@ -657,7 +657,7 @@ bool Interface::Parse(Json::Value *pChat)
 		Data *c = new Data;
 		c->bAFK = user.get("AFK", false).asBool();
 		string str = user.get("Name", "").asString();
-		AsciiToUnicode(str.c_str(), c->sender);
```

Comment:
Currently the forum will be sending only ASCII, but as I understand it, this will continue to work just fine, and will work if we ever convert the forum to use UTF8 for usernames and other text.  Correct?

Reply:
@schikore Yes, that's exactly right.  These changes should forward-proof us against ever wanting to start using UTF8 in these contexts.